upstream-seed/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "upstream-seed"
+version = "0.1.0"
+license = "GPL-3.0-only"
+authors = ["radicle team <dev@radicle.xyz>"]
+edition = "2018"
+description = "Minimal seed node for working with upstream"
+readme = "https://github.com/radicle-dev/radicle-upstream/blob/master/proxy/README.md"
```

Comment:
This is a 404.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2565/commits/9962818b84b32c2cabe0f6ae8f18a27dc35da124